recipes/colormap/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "colormap" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "eb73ca956770ae4e92a071d53daeb3c20d02c8b167797a2784f739db09749edd" %}
+
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/dd/de/2ab8312cdf4d4c2bc59b6fef4c51bf616dd431251d858079678c84c6359e/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+    - matplotlib
+    - easydev
+
+test:
+  imports:
+    colormap
+
+about:
+  home: http://packages.python.org/colormap/
+  license: GPL
```

Comment:
Isn't this LGPL? Also please include the version. Can also add `license_family: LGPL` if you wish a more general field like this one.